aclint/src/ACLINT.scala

```diff
@@ -52,10 +58,15 @@ class ACLINTInterface(parameter: ACLINTParameter) extends Bundle {
     Flipped(axi4.bundle.verilog.irrevocable(parameter.axi4parameter)).asInstanceOf[AXI4RWIrrevocableVerilog]
   val mtimerCompare =
     Flipped(axi4.bundle.verilog.irrevocable(parameter.axi4parameter)).asInstanceOf[AXI4RWIrrevocableVerilog]
-  val mip = Output(Vec(parameter.harts, Bool()))
+  val mswiDevice =
+    Flipped(axi4.bundle.verilog.irrevocable(parameter.axi4parameter)).asInstanceOf[AXI4RWIrrevocableVerilog]
+  val sswiDevice =
+    Flipped(axi4.bundle.verilog.irrevocable(parameter.axi4parameter)).asInstanceOf[AXI4RWIrrevocableVerilog]
+  val mip = Output(new ACLINTCsr(parameter))
+  val sip = Output(new ACLINTCsr(parameter))
```

Comment:
I think you just need 
```suggestion
  val mip = Output(Vec(parameter.harts, Bool()))
  val tip = Output(Vec(parameter.harts, Bool()))
```

Reply:
See https://github.com/chipsalliance/rocket-chip/blob/d0c6b50fdefcdbe121e9788433ea51f7efaf1d32/src/main/scala/devices/tilelink/CLINT.scala#L80-L84